docs/README.md

```diff
@@ -142,10 +146,6 @@ See [manual-instrumentation.md](manual-instrumentation.md).
 
 See [troubleshooting.md](troubleshooting.md).
 
-## Contributing
```

Comment:
Should we keep this section in addition to the callout that you moved under "We need you"?

Reply:
Reverted https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/575/commits/3184584c84e984d551f786158e1a8e0b349a7ce6